Libraries/acdriver/Sources/CompileAction.cpp

```diff
@@ -239,12 +239,25 @@ WarnUnsupportedOptions(Options const &options, Result *result)
     }
 }
 
+static bool
+AreOptionsValid(Options const &options, Result *result)
```

Comment:
Why isn't this a member function in the `Options` class?

Reply:
it could be :D